docs/manual/mkdocs.yml

```diff
@@ -227,15 +226,62 @@ nav:
       - project:
         - project upgrade: 'cmd/spfx/project/project-upgrade.md'
     - Microsoft Graph (graph):
+      - schemaextension:
+        - schemaextension add: 'cmd/graph/schemaextension/schemaextension-add.md'
+    - Outlook:
+        - outlook sendmail: 'cmd/outlook/outlook-sendmail.md'
+    - Microsoft Teams:
+      - teams app install: 'cmd/teams/teams-app-install.md'
+      - teams app list: 'cmd/teams/teams-app-list.md'
+      - teams app publish: 'cmd/teams/teams-app-publish.md'
+      - teams app remove: 'cmd/teams/teams-app-remove.md'
+      - teams app uninstall: 'cmd/teams/teams-app-uninstall.md'
+      - teams app update: 'cmd/teams/teams-app-update.md'
+      - teams channel add: 'cmd/teams/teams-channel-add.md'
+      - teams channel get: 'cmd/teams/teams-channel-get.md'
+      - teams channel list: 'cmd/teams/teams-channel-list.md'
+      - teams channel set: 'cmd/teams/teams-channel-set.md'
+      - teams funsettings list: 'cmd/teams/teams-funsettings-list.md'
+      - teams guestsettings list: 'cmd/teams/teams-guestsettings-list.md'
+      - teams guestsettings set: 'cmd/teams/teams-guestsettings-set.md'
+      - teams membersettings list: 'cmd/teams/teams-membersettings-list.md'
+      - teams membersettings set: 'cmd/teams/teams-membersettings-set.md'
+      - teams message get: 'cmd/teams/teams-message-get.md'
+      - teams message list: 'cmd/teams/teams-message-list.md'
+      - teams membersettings list: 'cmd/teams/teams-membersettings-list.md'
+      - teams messagingsettings list: 'cmd/teams/teams-messagingsettings-list.md'
+      - teams messagingsettings set: 'cmd/teams/teams-messagingsettings-set.md'
+      - teams report deviceusageuserdetail: 'cmd/teams/teams-report-deviceusageuserdetail'
+      - teams tab list: 'cmd/teams/teams-tab-list.md'
+      - teams user add: 'cmd/graph/o365group/o365group-user-add.md'
+      - teams user list: 'cmd/graph/o365group/o365group-user-list.md'
+      - teams user remove: 'cmd/graph/o365group/o365group-user-remove.md'
+      - teams user set: 'cmd/graph/o365group/o365group-user-set.md'
+      - teams team add: 'cmd/teams/teams-team-add.md'
+      - teams team archive: 'cmd/teams/teams-team-archive.md'
+      - teams team clone: 'cmd/teams/teams-team-clone.md'
+      - teams team list: 'cmd/teams/teams-team-list.md'
+      - teams team remove: 'cmd/teams/teams-team-remove.md'
+      - teams team set: 'cmd/teams/teams-team-set.md'
+      - teams team unarchive: 'cmd/teams/teams-team-unarchive.md'
+    - Flow:
+      - flow environment get: 'cmd/flow/flow-environment-get.md'
```

Comment:
Articles should be grouped by the area they operate on, eg. `environment`, `run`, etc. See existing `spo` commands for reference. Commands without an area `flow get`, `flow list`, should be on top.

Reply:
Done